### PR TITLE
Child model added

### PR DESCRIPTION
**Pull request by ben-gee723**

### CODE REVIEW

File: backend/model/childModel.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+const ChildSchema = new Schema({
+    firstName: { type: String, required: true },
+    lastName: { type: String, required: true },
+    birthday: { type: Date, min: "1950–01–01", required: true },
+    address: {
+        type: AddressSchema,
+        required: true
+    },
+    groupName: { type: String, required: true },
+    img: { type: String, required: true },
```
Comment (frausafiye) [inline backend/model/childModel.js]: img_required:false

File: backend/model/childModel.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    allergies: { type: String, required: true },
+    dietaryNeeds: { type: String, required: true },
+    // img: {type:String, required:true},
```
Comment (frausafiye) [inline backend/model/childModel.js]: no need that comment anymore,

File: backend/model/childModel.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+    // new Schema/Model for emergency contact?
+    emergencyContact: { type: String, required: true }
```
Comment (frausafiye) [inline backend/model/childModel.js]: emergencyContact:{type:[{name:String,address:AddressSchema,phoneNumber:String}],required:true } ??

File: backend/model/childModel.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    firstName: { type: String, required: true },
+    lastName: { type: String, required: true },
+    birthday: { type: Date, min: "1950–01–01", required: true },
+    address: {
+        type: AddressSchema,
+        required: true
+    },
+    groupName: { type: String, required: true },
+    img: { type: String, required: true },
+    allergies: { type: String, required: true },
```
Comment (frausafiye) [inline backend/model/childModel.js]: allergies:{type:[{to:String}],required:true } ??
dietaryNeeds:{type:[{requirement:String}],required:true } ??

File: backend/model/childModel.js (hunk 1 above)
Comment (ben-gee723) [reply backend/model/childModel.js]: Done!

File: backend/model/childModel.js (hunk 3 above)
Comment (ben-gee723) [reply backend/model/childModel.js]: Done

File: backend/model/childModel.js (hunk 4 above)
Comment (ben-gee723) [reply backend/model/childModel.js]: Done

**Pull request merged**